tests/baselines/reference/intTypeCheck.errors.txt

```diff
@@ -121,6 +123,8 @@ tests/cases/compiler/intTypeCheck.ts(205,17): error TS2351: Cannot use 'new' wit
     interface i6 extends i2 { }
     interface i7 extends i3 { }
     interface i8 extends i4 { }

```

Comment:
Since the goal of this test is not to test `i8` primitive types, it would be best to rename this interface and restore the prior test results (but with the newly renamed interface).